reactDashboard/src/components/Footer/Footer.jsx

```diff
@@ -98,7 +101,7 @@ const Footer = () => {
                   Proudly made by volunteers from
                   <br />
                   <img
-                    src="/STABanner.png"
+                    src="/STABanner2.png"
```

Comment:
If the new png is a straight replacement, bin the old one and rename the new one to STABanner.png